packages/@apphosting/adapter-angular/src/bin/build.ts

```diff
@@ -1,20 +1,87 @@
 #! /usr/bin/env node
 import { spawn } from "child_process";
-import { loadConfig } from "../utils.js";
+import { loadConfig, ApplicationBuilderOptions, move, writeFile } from "../utils.js";
+import { resolve, normalize, relative } from "path";
+import { stringify as yamlStringify } from "yaml";
 
+interface OutputBundleOptions {
+  bundleYamlPath: string;
+  outputDirectory: string;
+  outputBaseDirectory: string;
+  serverFilePath: string;
+  browserDirectory: string;
+}
+
+const cwd = process.cwd();
+
+// read Application Builder config
+const config: ApplicationBuilderOptions = await loadConfig(cwd);
+const outputPath = config.outputPath;
+// normalized output path structure
+const normalizedOutputPath = {
+  browser: "browser",
```

Comment:
When are we expecting that developers can override these? Should we build that in now?

Reply:
added manifest support